tests/integration/empty_repo_test.go

```diff
@@ -105,3 +109,32 @@ func TestEmptyRepoUploadFile(t *testing.T) {
 	resp = session.MakeRequest(t, req, http.StatusOK)
 	assert.Contains(t, resp.Body.String(), "uploaded-file.txt")
 }
+
+func TestEmptyRepoAddFileByAPI(t *testing.T) {
+	defer tests.PrepareTestEnv(t)()
+
+	err := user_model.UpdateUserCols(db.DefaultContext, &user_model.User{ID: 30, ProhibitLogin: false}, "prohibit_login")
```

Comment:
Can't you simply use another user?

Reply:
I do not want to bloat the test fixtures. There are sooo many strange users/repos.

Reply:
user1 user2  are already used everywhere and a lot.

I do not think it's right to add user100 user999 user99999 in the future.

Reply:
In fact, I have some plans to remove some unnecessary test fixtures, eg: the newly added user34 ......

Reply:
Could probably also add a shared user creation function that generated unique usernames (uuid etc.) with auto-assigned id.

Reply:
Yeah, and tests should clean up after themeselves, e.g. delete created resources in both success and failure case 😆

Reply:
Sure, after each test, all the data is reset, so this `UpdateUserCols` is 100% safe and right.